packages/engine/src/scene/components/ColliderComponent.ts

```diff
@@ -177,7 +166,7 @@ export const ColliderComponent = defineComponent({
     const groupComponent = useOptionalComponent(entity, GroupComponent)
 
     useLayoutEffect(() => {
-      SceneAssetPendingTagComponent.removeResource(entity, ColliderComponent.jsonID)
+      setComponent(entity, InputComponent)
```

Comment:
Pointing out as it's out of the norm. Assuming we're making an assumption that we want Colliders to always coexist with Input.

Reply:
Yeah we need to figure something out here, probably a reactor to do this instead of in the deserializer